nb-engine/nb-engine-cli/src/main/java/io/nosqlbench/engine/cli/NBCLI.java

```diff
@@ -136,12 +136,12 @@ public Integer apply(final String[] args) {
                     break;
                 }
 
-            final String error = NBCLIErrorHandler.handle(e, showStackTraces, NBCLI.version);
+            final int result = NBCLIErrorHandler.handle(e, showStackTraces, NBCLI.version);
             // Commented for now, as the above handler should do everything needed.
-            if (null != error) System.err.println("Scenario stopped due to error. See logs for details.");
```

Comment:
It seems like telling the user there was an error is important. Why remove this detail?

Reply:
I can adjust this.